query-service-impl/build.gradle.kts

```diff
@@ -31,6 +31,9 @@ dependencies {
           "in org.jetbrains.kotlin:kotlin-stdlib@1.4.10"
       )
     }
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.13.4.2") {
```

Comment:
this is likely constrained to a higher version in other projects - if so, should, match here. In other words, after removing this, go to the `query-service` dir, run `gw dependencyInsight --dependency jackson-databind --configuration runtimeClasspath` and see what version resolves there, then use that.

The point is that we'd rather compile with the version we'll end up running with.

Reply:
> Task :query-service:dependencyInsight
com.fasterxml.jackson.core:jackson-databind:2.14.2

So changed to 2.14.2